src/pages/dashboard/home.tsx

```diff
@@ -1,20 +1,84 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
+import styled from 'styled-components';
 import {useHistory} from 'react-router-dom';
-import {useAuth} from '../../hooks';
+import {useAuth, useApplication} from '../../hooks';
+import {Button, LoadingSymbol} from '../../components';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  min-width: 100vw;
+  min-height: 100vh;
+`;
+
+const StyledButton = styled(Button)`
+  margin-top: 16px;
+`;
 
 export default () => {
   const history = useHistory();
   const {signOut} = useAuth();
+  const {isLoading, application} = useApplication();
 
-  const handleClick = async () => {
+  const handleSignOut = async () => {
     await signOut();
     history.push('/');
   };
 
+  if (isLoading) {
+    return (
+      <Container>
+        <p>Let's see here...</p>
+        <LoadingSymbol />
+      </Container>
+    );
+  }
+
+  if (application === null) {
+    return (
+      <Container>
+        <h1>Register now!</h1>
+        <p>Don't wait! Start registration by pressing the button below.</p>
+        <StyledButton
+          kind='link'
+          color='var(--wine)'
+          link='/dashboard/registration'
+        >
+          Start Registration
+        </StyledButton>
+        <StyledButton
+          kind='button'
+          color='var(--wineDark)'
+          action={handleSignOut}
+        >
+          Sign Out
+        </StyledButton>
+      </Container>
+    );
+  }
+
   return (
-    <div>
-      <h1>Dashboard Home!</h1>
-      <button onClick={handleClick}>Sign Out</button>
-    </div>
+    <Container>
+      <h1>All set!</h1>
+      <p>You're good to go. Join our Discord Server for further updates.</p>
```

Comment:
We can re-emphasize the use of discord _during_ the actual event here as well. Some people may not have a discord account yet and may need extra motivation to create an account.

Reply:
Will make this change in #40 since that's where this is added initially. How does this sound:
```html
<h1>Almost done...</h1>
<p>The hackathon will take place right here and on Discord. Be sure to join our server so you don't miss out!</p>
```

Reply:
We should still _also_ emphasize that's where updates will be sent out.

Reply:
How about
```html
<h1>Almost done...</h1>
<p>
    The hackathon will take place over Discord alongside this website, and
    it's where we'll be sending frequent updates leading up to the event.
    <br />
    <br />
    Join now so you don't miss out!
</p>
```

Reply:
```html
<h1>Almost done...</h1>
<p>
    The hackathon will take place on our Discord along with this website.
    We'll be sending frequent updates leading up to the event there.
    <br />
    <br />
    Join now so you don't miss out!
</p>
```